.github/CODEOWNERS

```diff
@@ -38,5 +38,7 @@ crates/wascap @thomastaylor312 @connorsmith256 @brooksmtownsend @autodidaddict
 
 # projects
 crates/providers @wasmCloud/capability-provider-maintainers
+crates/wash-cli @autodidaddict @brooksmtownsend @thomastaylor312 @mattwilkinsonn
+crates/wash-lib @autodidaddict @brooksmtownsend @thomastaylor312 @mattwilkinsonn
```

Comment:
TODO: update to `wash-maintainers`

Reply:
Please open a PR, I have no control over the teams

Reply:
https://github.com/wasmCloud/wasmCloud/pull/829